examples/storage-examples/ceph-examples/mysql_ceph_pvc/ceph-claim.yaml

```diff
@@ -0,0 +1,10 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: ceph-claim
+spec:
+  accessModes:
+    - ReadWriteMany
```

Comment:
It is odd `ReadWriteMany` works in your example, rbd doesn't support it.
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/rbd/rbd.go#L71